eclipsecs-sevntu-plugin/src/com/github/sevntu/checkstyle/checks/coding/checkstyle-metadata.xml

```diff
@@ -179,6 +179,19 @@
       <message-key key="forbid.certain.imports"/>
     </rule-metadata>
 
+    <rule-metadata name="%ForbidCertainMethodCheck.name" internal-name="ForbidCertainMethodCheck" parent="TreeWalker">
+      <alternative-name internal-name="com.github.sevntu.checkstyle.checks.coding.ForbidCertainMethodCheck"/>
+      <description>%ForbidCertainMethodCheck.desc</description>
+      <property-metadata name="methodName" datatype="Regex" default-value="exit">
```

Comment:
after testing report is it is clear that default value should be "System.exit", just "exit" is too broad and make a false positives.
please change in all places.

Reply:
We can match only on method name and not on the class/object to which method belong.

See: https://github.com/checkstyle/checkstyle/issues/5083#issuecomment-327979559